bundle/manifests/metallb-operator.clusterserviceversion.yaml

```diff
@@ -479,8 +479,7 @@ spec:
   version: 0.0.0
   webhookdefinitions:
     - admissionReviewVersions:
-        - v1
```

Comment:
just wondering, why did you remove v1?

Reply:
sorry missed the next change

Reply:
make manifests was the one trigger the version changes as well as the change in deepcopy below